### PR TITLE
Fully remove dependency on grpc-gateway

### PR DESCRIPTION
**Pull request by pavelkalinnikov**
It was removed in #1677.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -250,9 +249,8 @@ github.com/grpc-ecosystem/go-grpc-middleware v1.2.2 h1:FlFbCRLd5Jr4iYXZufAvgWN6A
 github.com/grpc-ecosystem/go-grpc-middleware v1.2.2/go.mod h1:EaizFBKfUKtMIF5iaDEhniwNedqGo9FuLFzppDr3uwI=
 github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0 h1:Ovs26xHkKqVztRpIrF/92BcuyuQ/YW4NSIpoGtfXNho=
 github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0/go.mod h1:8NvIoxWQoOIhqOTXgfV/d3M/q6VIi02HzZEHgUlZvzk=
+github.com/grpc-ecosystem/grpc-gateway v1.9.5 h1:UImYN5qQ8tuGpGE16ZmjvcTtTw24zw1QAp/SlnNrZhI=
```
Comment (Martin2112) [inline go.sum:252]: Do you know what's pulling this in?

**Review by Martin2112 — APPROVED** [review]
One less build problem to worry about.

File: go.sum (hunk 1 above)
Comment (pavelkalinnikov) [reply go.sum:252]: Yes, it's etcd.

Comment (Martin2112) [reply go.sum:252]: OK, guess we're stuck with it then.

**Pull request merged**